lib/systems/include/Controllers/SimpleLaunchController.h

```diff
@@ -28,9 +28,10 @@ class TorqueControllerSimpleLaunch : public virtual BaseLaunchController
         : BaseLaunchController(initial_speed_target),
           launch_rate_target_(launch_rate) {}
 
-    
+    /// @brief base constrcutor with default values: Default_Launch_Rate = 11.76, DEFAULT_LAUNCH_SPEED_TARGET = 1500(rpm)
     TorqueControllerSimpleLaunch() : TorqueControllerSimpleLaunch(SLParams::DEFAULT_LAUNCH_RATE, SLParams::DEFAULT_LAUNCH_SPEED_TARGET) {}
-
+    /// @brief Increases speed target during launch linearly
```

Comment:
Again, I'm not clear on what calc_launch_algo is. And if I'm not clear on it, then this must be described more thoroughly.

Reply:
im confused what ur confused on, are you aware of how the torque and speed of the car works?

Reply:
No need for snark. Also, if you're going to ask a question, please be precise. "Are you aware of how the torque and speed of the car works?" is unclear. How would you like me to answer it?

If you instead asked "Are you familiar with the structure of all LaunchControllers and which functions they're required to implement?" my answer would be "No. I'd like to learn how the code behaves from the **documentation** rather than the **implementation**."

Reply:
wasnttrying to be snarky, just trying to figure out what else I had to explain

Reply:
also just for clarification, the troque and speed setpoints are controlled by all the controllers, not just the launch controllers, I added this to both the base controller documentation and shared data types for clarity

Reply:
lemme know if theres any more confusion about how the torque/speed and the controllers interact due to lack of info in docs